### PR TITLE
DominikReichl.KeePass version 2.52

### PR DESCRIPTION
**Pull request by tsteward**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/79282)

### CODE REVIEW

File: manifests/d/DominikReichl/KeePass/2.52/DominikReichl.KeePass.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  InstallerUrl: https://sourceforge.net/projects/keepass/files/KeePass%202.x/2.52/KeePass-2.52.msi/download
+  InstallerSha256: 3EADEF4AC19E279878F2E48FEBF6FD830A66AFFEBFB2B59B8F36FC0A03BE0190
+  ProductCode: '{18E3CF53-E927-445A-9F34-79CD3F22ED2B}'
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://sourceforge.net/projects/keepass/files/KeePass%202.x/2.52/KeePass-2.52-Setup.exe/download
+  InstallerSha256: DA403BC2E91132D1C1E0C49F585441E4CD430C8195CA8AF38ADC2EA300DE52CB
+  ProductCode: KeePassPasswordSafe2_is1
+  AppsAndFeaturesEntries:
+  - DisplayName: KeePass Password Safe 2.51.1
```
Comment (OfficialEsco) [inline manifests/d/DominikReichl/KeePass/2.52/DominikReichl.KeePass.installer.yaml:34]:
```suggestion
  - DisplayName: KeePass Password Safe 2.52.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/d/DominikReichl/KeePass/2.52/DominikReichl.KeePass.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: DominikReichl.KeePass
+PackageVersion: 2.52
```
Comment (OfficialEsco) [inline manifests/d/DominikReichl/KeePass/2.52/DominikReichl.KeePass.installer.yaml:5]:
```suggestion
PackageVersion: 2.52.0
```

File: manifests/d/DominikReichl/KeePass/2.52/DominikReichl.KeePass.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: DominikReichl.KeePass
+PackageVersion: 2.52
```
Comment (OfficialEsco) [inline manifests/d/DominikReichl/KeePass/2.52/DominikReichl.KeePass.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.52.0
```

File: manifests/d/DominikReichl/KeePass/2.52/DominikReichl.KeePass.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: DominikReichl.KeePass
+PackageVersion: 2.52
```
Comment (OfficialEsco) [inline manifests/d/DominikReichl/KeePass/2.52/DominikReichl.KeePass.yaml:5]:
```suggestion
PackageVersion: 2.52.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Should do these too, you must edit the Version folder name too, you can do this by pressing `.` and changing the folder via VSCode Online

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**